package.json

```diff
@@ -6,16 +6,18 @@
   "license": "ISC",
   "scripts": {
     "start": "gulp",
-    "build": "gulp build"
+    "build": "cross-env NODE_ENV=production gulp build"
```

Comment:
That's the right way to assign`NODE_ENV`. `cross-env` package helps to make this declaration cross-platform because it's different for Windows, Linux-based OS and MacOS